### ISSUE TITLE
Experiences regarding feedback serial AD values. Trigger for VOUT?

### ISSUE THREAD

**Issue by LoveFlashSonoff** [MaxusDev/MWTool#9]
I got the unit working via the serial interface and got values back.
The parameters kept with the standard values (so i did not changed them) except debug=00002.

The serial values were all in range of 350 to 550 - but mostly there are values of 4xx - mainly 46x and 47x.
Any experiences of the value when motion is being detected.
Do the value change dependly the distance of the object?

What treshold has been choosen to trigger VOUT?

**Comment by 0x6d7a** [comment]
> I got the unit working via the serial interface and got values back.
> The parameters kept with the standard values (so i did not changed them) except debug=00002.
> 
> The serial values were all in range of 350 to 550 - but mostly there are values of 4xx - mainly 46x and 47x.
> Any experiences of the value when motion is being detected.
> Do the value change dependly the distance of the object?
> 
> What treshold has been choosen to trigger VOUT?

@LoveFlashSonoff Yes the value does change regarding to the distance of the object and also to the reflectivity of the object. It's more sensitive if the object is metallic. Moreover, it's also sensitive to your environment and the placement of the module. I suggest you try different value to test the module.
